ecosystem/sep-0038.md

```diff
@@ -0,0 +1,176 @@
+`decimals` | integer | The number of decimals needed to represent the currency.
+
+### GET Price
+
+This endpoint can be used to fetch the indicative price of a Stellar asset and currency [pair](https://en.wikipedia.org/wiki/Currency_pair).
+
+#### Request
+
+Name | Type | Description
+-----|------|------------
+`base` | string | The base asset or currency. Ex. `USDC:circle.io`, `ARS`
+`counter` | string | The counter asset or currency.
+`base_amount` | string | The amount of the base asset or currency to be traded for the counter asset or currency.
+
+#### Response
+
+Name | Type | Description
+-----|------|------------
+`price` | string | The price of one unit of the base asset or currency in terms of the counter asset or currency for the `base_amount` specified in the request.
+
+### POST Quote
+
+This endpoint can be used to request a [firm](https://www.investopedia.com/terms/f/firmquote.asp) quote for a Stellar asset and currency pair. 
+
+In contrast with the `GET price(s)` endpoints, the amount requested must be held in reserve and not used in calculations of subsequent quotes until the expiration provided in the response.
+
+#### Protecting Against Bad Actors
+
+To protect against bad actor clients reserving all available captial without following through with the requested trades, servers should only accept requests for which the following is true:
+
+1. The Stellar account authenticated via SEP-10 is owned by a known and trusted entity, or
+2. The owner of the Stellar account has been properly KYC'ed, either via [SEP-12](sep-0012.md) or some other mechanism.
+
+Entities or individuals that misuse the service can be placed on a denylist that should be checked before processing requests for quotes.
+
+#### Transaction Fees
+
+It should be expected that the provided quote includes a margin for the provider as a service fee, and this margin will vary depending on the directional flow of funds.
+
+If the client requests some amount of off-chain currency for providing some amount of a Stellar asset, the client will submit a Stellar transaction delievering funds to the anchor before the expiration included in the response, paying a fee as determined by state of the Stellar network. 
+
+In the reverse scenario, the anchor will submit a Stellar transaction to deliver funds to the client as long as the client delivered off-chain funds to the anchor before the expiration. In this case, the anchor will likely increase their margin to cover the cost of submitting the transaction.
+
+This specification could be extended in the future to include functionality where the anchor collects a transaction signature  from the client prior to submitting the transaction, enabling the anchor to have the client pay the network fees and decrease their margin. This feature would increase the technical complexity of the specification and hence will only be added if there is enough support from the ecosystem.
```

Comment:
💡 I don't think it is worth including the paragraph about some possible extension, as it distracts from the rest of the content in this section. I don't feel strongly about this, just my perspective. Businesses pay fees to move money, it's part of doing business, and the fees are likely larger for the off-chain movements of money, so I don't think we should encourage making the UX and process more complex for it.